desmod/simulation.py

```diff
@@ -140,7 +142,20 @@ def __enter__(self):
             os.chdir(self.workspace)
 
     def __exit__(self, *exc):
-        os.chdir(self.prev_dir)
+        os.chdir(self.orig_dir)
+        try:
+            os.chdir(self.config['sim.workspace'])
+            self.sync()
+        finally:
+            os.chdir(self.orig_dir)
+
+    def sync(self):
+        if self.config['sim.workspace.s3_sync']:
```

Comment:
My bias is to use `sim.sync.s3.` as the base namespace. In addition to being decoupled from `sim.workspace`, it also ends up being less typing and a shallower hierarchy--both positives.